images/alt.yaml

```diff
@@ -182,5 +182,17 @@ actions:
   variants:
   - cloud
 
+# XXX: Prevent cloud-init from attempting to modify the hostname. Otherwise, cloud-init fails with
+# an attribute error: "module 'cloudinit.util' has no attribute 'load_file'") in "update_hostname".
+- trigger: post-packages
+  action: |-
+    #!/bin/sh
+    set -eux
+
+    # Disable the update_hostname module in cloud-init.
+    sed -i '/update_hostname/d' /etc/cloud/cloud.cfg
```

Comment:
I wanted to find a way to drop a `/etc/cloud/cloud.cfg.d/` snippet to override that but AFAICT, that's not possible or at least not easily done while avoiding to redefine the `cloud_init_modules` list.